akka-remote/src/main/scala/akka/remote/RemoteWatcher.scala

```diff
@@ -32,7 +32,7 @@ private[akka] object RemoteWatcher {
   final case class UnwatchRemote(watchee: InternalActorRef, watcher: InternalActorRef)
 
   @SerialVersionUID(1L) case object Heartbeat extends HeartbeatMessage
-  @SerialVersionUID(1L) final case class HeartbeatRsp(addressUid: Int) extends HeartbeatMessage
+  @SerialVersionUID(1L) final case class HeartbeatRsp(addressUid: Long) extends HeartbeatMessage
```

Comment:
Yes, I cannot find any serialization dealing with HeartBeatRsp. But when I ran tests between 2.4.10 and this PR it didn't fail, I guess java serialization does the long->int conversion? Scary.

I'll try again with a changed serialversion uid to see if it fails then.